src/app/Dashboard/AutomatedAnalysis/AutomatedAnalysisCard.tsx

```diff
@@ -0,0 +1,724 @@
+        clearAllFilters={handleClearFilters}
+        clearFiltersButtonText="Clear"
+        isFullHeight
+      >
+        <ToolbarContent>
+          <AutomatedAnalysisFilters
+            target={targetConnectURL}
+            evaluations={categorizedEvaluation}
+            filters={targetAutomatedAnalysisFilters}
+            updateFilters={updateFilters}
+          />
+          <ToolbarGroup>
+            <ToolbarItem>
+              <Button
+                isSmall
+                isAriaDisabled={isLoading || usingCachedReport || usingArchivedReport}
+                aria-label="Refresh automated analysis"
+                onClick={generateReport}
+                variant="control"
+                icon={<Spinner2Icon />}
+              />
+              <Button
+                isSmall
+                isAriaDisabled={isLoading || usingArchivedReport}
+                aria-label="Delete automated analysis"
+                onClick={clearAnalysis}
+                variant="control"
+                icon={<TrashIcon />}
+              />
+            </ToolbarItem>
+            <ToolbarItem>
+              <Checkbox
+                label="Show N/A scores"
+                isChecked={showNAScores}
+                onChange={handleNAScoreChange}
+                id="show-na-scores"
+                name="show-na-scores"
+              />
+            </ToolbarItem>
+          </ToolbarGroup>
+        </ToolbarContent>
+      </Toolbar>
+    );
+  }, [
+    isLoading,
+    isError,
+    showNAScores,
+    targetConnectURL,
+    categorizedEvaluation,
+    targetAutomatedAnalysisFilters,
+    generateReport,
+    handleClearFilters,
+    handleNAScoreChange,
+    updateFilters,
+  ]);
+
+  const errorView = React.useMemo(() => {
+    return (
+      <ErrorView
+        title={'Automated Analysis Error'}
+        message={
+          <TextContent>
+            <Text component={TextVariants.p}>Cryostat was unable to generate an automated analysis report.</Text>
+            <Text component={TextVariants.small}>{errorMessage}</Text>
+          </TextContent>
+        }
+        retryButtonMessage={handleErrorView()[0]}
+        retry={handleErrorView()[1]}
+      />
+    );
+  }, [errorMessage, handleErrorView]);
+
+  const view = React.useMemo(() => {
+    if (isError) {
+      if (errorMessage == 'Authentication failure') {
+        return errorView;
+      }
+      return <AutomatedAnalysisConfigDrawer onCreate={generateReport} drawerContent={errorView} />;
+    } else if (isLoading) {
+      return <LoadingView />;
+    } else {
+      return filteredCategorizedLabels;
+    }
+  }, [filteredCategorizedLabels, isError, isLoading, errorMessage, errorView, generateReport]);
+
+  return (
+    <Card id="automated-analysis-card" isRounded isCompact isExpanded={isCardExpanded}>
+      <CardHeader
+        isToggleRightAligned
+        onExpand={onCardExpand}
+        toggleButtonProps={{
+          id: 'toggle-button1',
```

Comment:
should have unique id for this card.

Reply:
For the card or for the toggleButton? The card does have an id="automated-analysis-card".

Reply:
oh the id of the toggleButton :D